src/Notifier.php

```diff
@@ -11,27 +11,43 @@
 
 class Notifier
 {
-    public static function sendPush(string $heading, string $content, array $player_ids, array $extra = null)
+    private static $expire_at = 0;
+
+    private static $options = [];
+
+    public static function sendPush(string $heading, string $content, array $player_ids, array $extra = NULL)
     {
-        dispatch(new SendPushJob($heading, $content, $player_ids, $extra));
+        dispatch(new SendPushJob($heading, $content, $player_ids, $extra, static::$expire_at, static::$options));
```

Comment:
in static class in oop solution, if you need to select or use static variable you need to use self::VariableName
and if you need to create static object of class need to Write static::